src/openrct2/interface/Screenshot.cpp

```diff
@@ -21,21 +21,13 @@
 #include "../core/File.h"
 #include "../core/Imaging.h"
 #include "../core/Path.hpp"
-#include "../drawing/Drawing.h"
 #include "../drawing/X8DrawingEngine.h"
-#include "../localisation/Formatter.h"
```

Comment:
Do not remove includes that are used, even if the file compiles fine (since another header probably includes it). Formatter is used in `screenshot_giant`, so this header should definitely remain here. This goes for many other includes that are removed now too.

Reply:
Weird, the IDE tells me that these are unused and can be removed safely. Also compiled just fine.
If you tell me so then I put these includes back in. Except for Paint.h

Reply:
It compiles fine because these headers are included by other headers, but by removing them here you rely on them staying in the other headers. If they are removed from these headers, you suddenly get compiler errors in this file even when nothing is changed here.